playground/nextjs/pages/survey.tsx

```diff
@@ -29,6 +29,7 @@ export default function SurveyForm() {
     return (
         <>
             <div className="flex items-center gap-2 flex-wrap">
+                <button className="survey-feedback-button">Feedback</button>
```

Comment:
can we move this to a const, we are repeatin

Reply:
this is just on our internal nextjs playground, imo not needed as it's just the class i used for my survey. I left the code just because it might be helpful to other people in the future

Reply:
true that, my bad, i just did a search and saw the same className across places